src/rpdk/core/project.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, overwrite_enabled=False, root=None):
         self.settings = None
         self.schema = None
         self._flattened_schema = None
+        self._marked_down_properties = {}
```

Comment:
I think it's a bit strange this is part of the init rather than just passed in the recursive function since now that function is setting and keeping state. It's just another extra thing we have to be mindful in the future